### DISCUSSION TITLE
Idea: removing staff's patrol zones by right-clicking

### DISCUSSION THREAD

**Discussion by MASHzj** [OpenRCT2/OpenRCT2#20000] [Feature requests]
Since it's possible to zone staff by individual tiles instead of fixed 4x4 squares, it's sometimes unintuitive whether you are adding or removing to patrol zones depending on where you click. For example, if all tiles in your selection square are zoned, except the one your mouse is pointing at, that one tile is added instead of the rest removed. So I thought it might be a good idea to enable right-clicking for removing zones, in analogy with building. That would make it that left-clicking always adds patrol zone in your selection, and right-clicking removes it.

**Comment by J-PIE-314** [comment]
This would interfere with the option to scroll viewports on right-click.

**Reply by MASHzj** [reply]
I don't really see what the difference would be with the current behaviour when you have a building tool active: single right click to remove things and right dragging for scrolling.
